spec/features/pets/pets_create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe "pets", type: :feature do
```

Comment:
right now i', delineating my tests by the controller and each crud functionality or page. So create_spec is also testing the new html page